google/cloud/forseti/common/util/file_uploader.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility functions for file uploads."""
+
+
+import tempfile
+
+from google.cloud.forseti.common.data_access import csv_writer
+from google.cloud.forseti.common.gcp_api.storage import StorageClient
+from google.cloud.forseti.common.util import logger
+from google.cloud.forseti.common.util import parser
+
+
+LOGGER = logger.get_logger(__name__)
+
+
+def upload_json(data, gcs_upload_path):
+    """Upload inventory summary/violation data in json format.
+
+    Args:
+        data (dict): the data to upload
+        gcs_upload_path (string): the GCS upload path.
+    """
+    with tempfile.NamedTemporaryFile() as tmp_data:
+        tmp_data.write(parser.json_stringify(data))
+        tmp_data.flush()
+        storage_client = StorageClient()
+        storage_client.put_text_file(tmp_data.name, gcs_upload_path)
```

Comment:
What is the file name that we are using here?

Reply:
What do you mean?

Reply:
Ah!

    $ gsutil ls gs://fs-opendata/iv/
    gs://fs-opendata/iv/inv_summary.2018-04-15T17:20:42.382171.20180415T174457Z.csv
    gs://fs-opendata/iv/inv_summary.2018-04-15T17:20:42.382171.20180416T072300Z.json

Reply:
qq: how are we setting `tmp_data.name` to be `inv_summary` here?  Also, I like to avoid abbreviation in the actual filename output.  So, please make this `inventory_summary`.  Also, where are there 2 fragments of timestamps in the file name?  Can we just keep one?